virttest/virt_vm.py

```diff
@@ -1231,8 +1239,11 @@ def print_guest_network_info():
 
         print_guest_network_info()
         if serial:
-            return self.wait_for_serial_login(timeout, internal_timeout,
-                                              False, username, password)
+            session = self.wait_for_serial_login(timeout, internal_timeout,
+                                                 False, username, password)
+            if set_dumb_terminal:
+                session.cmd("export TERM=dumb")
```

Comment:
May I ask whether serial console is impacted by  set TERM=dumb too, or this is limited to login only?

Reply:
@chunfuwen The environment variable can only be set after log in.